File: py/3300.py
def suerte(n):
    for i in range(len(n) - 1):
        if n[i] == '1' and n[i + 1] == '3':
            return False

    return True


n = input()
print(n, 'NO ' * suerte(n) + 'es de Mala Suerte')

File: cpp/3300.cpp
#include <iostream>
#include <string>

using namespace std;

bool suerte(string n)
{
    int len = n.length(), i;
    len--;

    for (i = 0; i < len; i++) {
        if (n[i] == '1' && n[i + 1] == '3') {
            return false;
        }
    }

    return true;
}

int main()
{
    string number;
    cin >> number;

    if (suerte(number)) {
        cout << number << " NO es de Mala Suerte\n";
    } else {
        cout << number << " es de Mala Suerte\n";
    }

    return 0;
}